### PR TITLE
Braintree payment form

### PR DESCRIPTION
**Pull request by rjmackay**
Refs ushahidi/platform-cloud-interface#113

This change is [Reviewable](https://reviewable.io/reviews/ushahidi/platform-pattern-library/34)

### CODE REVIEW

File: pattern-library/5_layouts/tour.html [hunk 1]
```diff
@@ -281,12 +269,95 @@ <h2 class="tour-heading">We need just a few moments to finish building your depl
...
+            },
+            expirationYear: {
+                selector: "#cc-exp-year",
+                placeholder: "YYYY"
+            },
+            postalCode: {
+                selector: "#cc-zip"
+            },
+            styles: {
+                "input": {
```
Comment (rjmackay) [inline pattern-library/5_layouts/tour.html]: We don't get Lato because of the iframes - so I've had to tweak font size a bit to try and match

**Comment by middle8media** [comment]
Unable to focus the inputs in Chrome. Works fine in FF.

Other than that, the styles look good.

**Comment by rjmackay** [comment]
> Unable to focus the inputs in Chrome

That is super weird since I was developer in Chrome. I can't reproduce that myself.. can you retest and see if its an intermittent issue or always broken for you? or if you can figure out *why* its broken. Otherwise I'm stuck :(

---

Review status: 0 of 23 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/ushahidi/platform-pattern-library/34#-:-KGjUj5GrflouxC2bjG8:1389633359)*

**Comment by middle8media** [comment]
Just double checked, all good, ignore this, Chrome acts buggy for me sometimes.

**Pull request merged**